File: core/schema-discovery/schema-discovery-api/src/main/java/com/thinkbiganalytics/discovery/schema/TableSchema.java
package com.thinkbiganalytics.discovery.schema;

/**
 * Schema of a table object
 */
public interface TableSchema extends Schema {

    /**
     * Name of the table schema
     */
    String getSchemaName();

    /**
     * Database name
     */
    String getDatabaseName();

}

File: ui/ui-app/src/main/resources/static/js/feed-mgr/model/table-schema.ts
import {Schema} from "./schema";
import {SchemaField} from "./schema-field";

/**
 * Schema of a table object
 */
export interface TableSchema extends Schema {

    /**
     * Table schema name
     */
    schemaName: string;

    /**
     * Database name
     */
    databaseName: string;


}
